src/expandable-section/expandable-section-header.tsx

```diff
@@ -135,7 +135,7 @@ const ExpandableHeaderTextWrapper = ({
 }: ExpandableHeaderTextWrapperProps) => {
   const isContainer = variant === 'container';
   const HeadingTag = headingTagOverride || 'div';
-  const supportsInteractiveElements = variantSupportsInteractiveElements(variant);
+  const supportsInteractiveElements = variantSupportsActions(variant);
```

Comment:
NIT: 
```suggestion
  const supportsActions= variantSupportsActions(variant);
```

Reply:
Well, the variable is technically still used to determine if it supports interactive elements (including info links and actions). I just didn't want to overcomplicate this too much by adding three new utils.